man/scrot.txt

```diff
@@ -3,7 +3,7 @@ NAME
 
 SYNOPSIS
   scrot [-bcfhkmopsuvz] [-a X,Y,W,H] [-C NAME] [-D DISPLAY] [-d SEC] [-e CMD]
-        [-l STYLE] [-n OPTS] [-q NUM] [-S CMD] [-t NUM | GEOM] [FILE]
+        [-f FILE] [-l STYLE] [-n OPTS] [-q NUM] [-S CMD] [-t NUM | GEOM] [FILE]
```

Comment:
It must be capitalized: `-F`

Reply:
Thanks! I'll fix that. It's a typo, I meant to capitalize it.